stdlib/2and3/shutil.pyi

```diff
@@ -20,18 +21,19 @@ from typing import (
 
 if sys.version_info >= (3, 6):
     _AnyStr = str
-    _AnyPath = TypeVar("_AnyPath", str, os.PathLike[str])
+    _AnyPath = TypeVar("_AnyPath", bytes, str, os.PathLike[str])
```

Comment:
Question for the project maintainers:

Should `_AnyPath` be dropped in favor of `_typeshed.AnyPath`? If so, I'm happy to go ahead with such a change.

Same question applies to `_AnyStr` and `typing.AnyStr`.

Reply:
They're semantically different, though in confusing ways: `_AnyPath` is a TypeVar and `_typeshed.AnyPath` is a Union, while `_AnyStr` is an alias and `AnyStr` is a TypeVar.